code/modules/projectiles/projectile/beams.dm

```diff
@@ -56,7 +56,7 @@
 	damage = 15
 	irradiate = 30
 	range = 15
-	pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE | PASSCLOSEDTURF
+	pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE | PASSCLOSEDTURF | PASSBLOB | PASSMOB

```

Comment:
An interesting caveat with these passflags is that they can go through walls but not computers. If you want to just hulk out the xray feel free to add `| PASSOBJ`

Reply:
i might consider it but also reduce the range and damage

Reply:
i checked the bitflags and passobj doesn't exist

Reply:
should probably have LETPASSTHROW though

Reply:
oh I must have made that in a working branch that I never PR'd

epic lulz

Reply:
@ShizCalev I tried LETPASSTHROW but the beams stopped at structures

Reply:
you'll have to make a new flag then.